APP/src/api.js

```diff
@@ -226,39 +226,39 @@ export const getLeaderboard = async (year, callback) => {
 
 export const postMatchScouting = async (data, callback) => {
   try {
-      const endpoint = `${API_ENDPOINT}/MatchScouting/`;
-      console.log(endpoint)
-      let retval;
-      let json = JSON.stringify(data)
-      const response = await fetch(endpoint, {
-        method: "POST", // *GET, POST, PUT, DELETE, etc.
-        headers: {
-          "Content-Type": "application/json",
-        },
-        body: JSON.stringify(data), // body data type must match "Content-Type" header
-      });
-      callback(response.status); // parses JSON response into native JavaScript objects
-  } catch (e){
+    const endpoint = `${API_ENDPOINT}/MatchScouting/`;
```

Comment:
Avoid including console.logs in your web app. Its helpful for debugging but shouldn't be seen by end users. consider Console.debug instead.